enterprise/frontend/src/embedding-sdk/cli/steps/show-post-setup-steps.ts

```diff
@@ -40,10 +47,27 @@ export const showPostSetupSteps: CliStepMethod = async state => {
   const POST_SETUP_STEPS = [];
 
   if (state.token) {
-    POST_SETUP_STEPS.push(STEP_1);
+    POST_SETUP_STEPS.push(START_SERVER_STEP);
+  }
+
+  if (!isNextJs) {
+    POST_SETUP_STEPS.push(REACT_COMPONENT_IMPORT_STEP);
+  }
```

Comment:
Only show the `import <AnalyticsPage />` step if we are not in Next.js, because for Next.js we had already generated the route files, so no manual import needed!